### ISSUE TITLE
Add Cellpose training

### ISSUE THREAD

**Issue by hermancollin** [axondeepseg/ASTIH#4]
Training Cellpose models is fairly straightforward, but the data needs some minor preprocessing. To streamline the whole process, here is my current plan:

1. Add preprocessing utilities to put the data in a format suitable for Cellpose training (takes as input a single-class instance segmentation, so we combine the axon and myelin masks together to form a "fiber" instance seg);
2. Preprocess all datasets using these utilities (inside the get_data.py script). This preprocessed data can be placed under `data/cellpose_pipeline/`;
3. Create a training script to use the same hyperparameters for all models;
4. Apply the trained models on the test sets;
5. Move the predictions into the `splits/test/` directory alongside the nnunet predictions;
6. Adapt the evaluation script so that it takes as input the instance segmentation output of Cellpose models (only for the object-detection part of the evaluation).
